modules/drivers/presto-jdbc/src/metabase/driver/presto_jdbc.clj

```diff
@@ -224,7 +224,9 @@
                                   (Integer/parseInt port)
                                   port)))
                 (assoc :SSL ssl?)
-                (dissoc :ssl))]
+                ;; remove Metabase specific properties that are not recognized by the PrestoDB JDBC driver, which is
+                ;; very picky about properties (throwing an error if any are unrecognized)
+                (dissoc :ssl :engine :let-user-control-scheduling))]
```

Comment:
if presto is very picky about keys, can we change this to a positive `select-keys` of what is valid for presto? So that any new keys in the future won't affect us and we play whack-a-mole? Probably not huge but perhaps a slightly more robust way to guard this.

Reply:
You're right; that would be a better approach.  I found the inclusive list from the source and added a new commit to do things that way instead.

Reply:
OK, it ended up being a fairly long key sequence, but in the end it's probably better/safer this way.

Reply:
yeah. and a bit more explicit so easier to have an idea of what is flowing through.